crates/vim/src/command.rs

```diff
@@ -1072,6 +1072,7 @@ fn generate_commands(_: &App) -> Vec<VimCommand> {
         VimCommand::new(("delm", "arks"), ArgumentRequired)
             .bang(DeleteMarks::AllLocal)
             .args(|_, args| Some(DeleteMarks::Marks(args).boxed_clone())),
+        // TODO: add SortLinesByLength here as well?
```

Comment:
TODO: handle this

Reply:
Thanks for thinkging about it, but no need for a vim alias (vim users can access all editor commands from the palette already, and this is not built into vim).

Reply:
Thanks @ConradIrwin - I just searched for uses of `SortLinesCaseSensitive` and `SortLinesCaseInsensitive` and vim came up here. Removed this comment now.